src/vm/functions/database.rs

```diff
@@ -51,7 +51,7 @@ pub fn special_fetch_variable(args: &[SymbolicExpression],
 
     let data = env.global_context.database.lookup_variable(&env.contract_context.name, var_name)?;
     match data {
-        Some(data) => Ok(Value::some(data)),
+        Some(data) => Ok(data),
         None => Ok(Value::none())
```

Comment:
This should be a new error type -- something like `InterpreterError::UninitializedPersistedVariable`

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1022/commits/eb0b857ed43a57709c70598fdd42105a52c1283a